client/src/utilities/treeItemUtilities.ts

```diff
@@ -0,0 +1,56 @@
+import { join } from 'path';
+
+import MezzuriteComponent from '../models/MezzuriteComponent';
+
+function generateDescription (fulfilled: boolean) {
+  let description = null;
+
+  if (fulfilled != null) {
+    if (fulfilled) {
+      description = 'Tracked';
+    } else {
+      description = 'Not Tracked';
+    }
+  }
+
+  return description;
+}
+
+function generateFulfilled (component: MezzuriteComponent): boolean {
+  let fulfilled = null;
+
+  if (component != null) {
+    if (Object.keys(component.checks).length > 0) {
+      fulfilled = Object.keys(component.checks)
+        .map((key: string) => component.checks[key])
+        .reduce((previousValue: boolean, currentValue: boolean) => {
+          return previousValue && currentValue;
+        });
+    } else {
+      fulfilled = true;
+    }
+  }
+
+  return fulfilled;
+}
+
+function generateIconPath (componentType: string, fulfilled: boolean, rootPath: string): string {
+  let iconPath = null;
+
+  if (componentType != null && fulfilled != null && rootPath != null) {
+    let iconKind = '';
+
+    if (!fulfilled) {
+      iconKind = 'Light';
+    }
+    iconPath = join(rootPath, 'client', 'res', 'icons', `${componentType + iconKind}.svg`);
```

Comment:
I'd recommend moving the template string out into its own variable for readability, just like in the previous comment. Although this one is easier to read than the previous one.

Reply:
Resolved via https://github.com/Microsoft/vscode-mezzurite/pull/33/commits/808368aeed44ebbcd418fe05e9542784865a8bc1.